example/server/integrations.py

```diff
@@ -783,77 +783,82 @@ def toml_integration():
-                        // email has been confirmed.
-                        window.location.reload(true);
-                    });
-                });
-            </script>
-            """
-        )
-        # Add a "Skip Confirmation" button that will make a GET request to the
-        # skip confirmation endpoint and reload the page. The email confirmation
-        # functionality is just for sake of demonstration anyway.
-        tags.append(
-            """
-            <script>
-                (function () {
-                    let section = document.querySelector(".main-content").firstElementChild;
-                    let button = document.createElement("button");
-                    button.className = "button";
-                    button.innerHTML = "Skip Confirmation";
-                    button.setAttribute("test-action", "submit");
-                    button.addEventListener("click", function () {
-                        this.disabled = true;
-                        let url = window.location.protocol + "//" + window.location.host + "/skip_confirm_email";
-                        fetch(url).then(res => res.json()).then((json) => {
-                            if (json["status"] === "not found") {
-                                // This would only happen if the PolarisStellarAccount doesn't exist.
-                                // It should always exist because the user needs to have an existing
-                                // account to access the confirm email page.
-                                let errElement = document.createElement("p");
-                                errElement.style = "color:red";
-                                errElement.innerHTML = "Error: Unable to skip confirmation step";
-                                errElement.align = "center";
-                                section.appendChild(document.createElement("br"));
-                                section.appendChild(errElement);
-                            } else {
-                                window.location.reload(true);
-                            }
-                        });
-                    });
-                    section.appendChild(document.createElement("br"));
-                    section.appendChild(button);
-                })();
-            </script>
-            """
-        )
+    tags = []
+    tags.append("sep24_scripts/test.js")
+
+    # ALEC TODO - uncomment and move to static folder
+    # tags = [
+    #     # Google Analytics
+    #     """
+    #     <!-- Global site tag (gtag.js) - Google Analytics -->
+    #     <script async src="https://www.googletagmanager.com/gtag/js?id=UA-53373928-6"></script>
+    #     <script>
+    #       window.dataLayer = window.dataLayer || [];
+    #       function gtag(){dataLayer.push(arguments);}
+    #       gtag('js', new Date());
+    #       gtag('config', 'UA-53373928-6');
+    #     </script>
+    #     """
+    # ]
+    # if (
```

Comment:
This logic still needs to be present in the integration function. We don't want to load this script unless its a particular page being served.